ocaml/xapi/xapi_vm_lifecycle.ml

```diff
@@ -168,6 +169,9 @@ let check_op_for_feature ~__context ~vmr ~vmgmr ~power_state ~op ~ref ~strict =
 			| `changing_VCPUs_live
 					when lack_feature "feature-vcpu-hotplug"
 						-> some_err Api_errors.vm_lacks_feature_vcpu_hotplug
+			| `suspend | `checkpoint | `pool_migrate | `migrate_send
```

Comment:
Yes, I checked that it worked, and also looked it up on Inria to check that it was officially specified: I wouldn't have wanted to rely on it if it had been unspecified behaviour that just happened to work in the current OCaml implementation.